certbot-apache/certbot_apache/tests/testdata/debian_apache_2_4/augeas_vhosts/apache2/sites-enabled/old-and-default.conf

```diff
@@ -0,0 +1,12 @@
+<VirtualHost *:80 [::]:80>
```

Comment:
It looks like this file got changed from a symlink to the file in `sites-available` to another copy of the file.

Reply:
Hm, I didn't copy the file, I made a symlink. I'll look into that.

Reply:
This sure is weird, it's definitely a copy of the file now

Reply:
Oh I forgot the `-s` flag haaaa